install/0000_30_machine-api-operator_00_credentials-request.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: cloudcredential.openshift.io/v1beta1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-cluster-api
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: aws-cloud-credentials
+    namespace: openshift-cluster-api
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1beta1
+    kind: AWSProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - ec2:DescribeImages
```

Comment:
this would need https://github.com/openshift/installer/blob/master/data/data/aws/master/main.tf#L48
```    {
      "Action": "iam:PassRole",
      "Resource": "*",
      "Effect": "Allow"
    },```
cc @spangenberg